examples/quickstart/run.py

```diff
@@ -74,22 +70,32 @@ def main(
 
     Args:
         model_type: Type of model to use
-        orchestration_environment: Environment where the pipeline will run
+        config_path: Configuration file to use
         no_cache: If `True` cache will be disabled.
     """
-    config_folder = os.path.join(
-        os.path.dirname(os.path.realpath(__file__)),
-        "configs",
-    )
-
+    client = Client()
     run_args_train = {}
 
+    crf = client.active_stack.container_registry.flavor
+    asf = client.active_stack.artifact_store.flavor
+    orchf = client.active_stack.orchestrator.flavor
+
     pipeline_args = {}
     if no_cache:
         pipeline_args["enable_cache"] = False
-    pipeline_args["config_path"] = os.path.join(
-        config_folder, f"training_{orchestration_environment}.yaml"
-    )
+
+    if not config_path:
+        if crf == "aws" and asf == "s3" and orchf == "sagemaker":
+            config_path = "configs/training_aws.yaml"
+        elif crf == "gcp" and asf == "gcp" and orchf == "vertex":
+            config_path = "configs/training_gcp.yaml"
+        elif crf == "azure" and asf == "azure" and orchf == "azureml":
+            config_path = "configs/training_azure.yaml"
+        else:
+            config_path = "configs/training_default.yaml"
+    print(f"Using {config_path} to configure the pipeline run.")
```

Comment:
While reviewing this, I had another idea: you should warn people if they use the aws/gcp/azure config files explicitly with a stack that doesn't match and if they haven't set `skip_build: False`. This will keep them from running into fuzzy errors later on because some integration is missing.